From 64f4eb79fe2977320660f8940bc908fa4def807b Mon Sep 17 00:00:00 2001
From: Daniel Golle <daniel@makrotopia.org>
Date: Wed, 1 Jan 2020 01:27:38 +0200
Subject: [PATCH] move resolv.conf.auto to /tmp/resolv.conf.d/

Using /tmp/resolv.conf.d/ has the advantage that we can mount-bind it.
Mount-bind'ing /tmp/resolv.conf.auto directly previously caused
problems as the file is being deleted/replaced at run-time.

Signed-off-by: Daniel Golle <daniel@makrotopia.org>
---
 netifd.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/netifd.h b/netifd.h
index e565423..a4a146c 100644
--- a/netifd.h
+++ b/netifd.h
@@ -37,7 +37,7 @@
 #define DEFAULT_MAIN_PATH	"/lib/netifd"
 #define DEFAULT_CONFIG_PATH	NULL /* use the default set in libuci */
 #define DEFAULT_HOTPLUG_PATH	"/sbin/hotplug-call"
-#define DEFAULT_RESOLV_CONF	"/tmp/resolv.conf.auto"
+#define DEFAULT_RESOLV_CONF	"/tmp/resolv.conf.d/resolv.conf.auto"
 #endif
 
 extern const char *resolv_conf;
-- 
2.30.0

